docs/faq.rst

```diff
@@ -124,10 +124,28 @@ Which ML libraries does Fairlearn support?
     `predict()` methods. Also, any classification or regression
     algorithm can be evaluated using our metrics.
 
+Does Fairlearn support multi-class classification?
+    Certain metrics will still work for multi-class classification, e.g.,
+    accuracy score. For others it is more complicated since they rely on
+    binary classes, e.g., true positive rate.
```

Comment:
How about

> The `MetricFrame` does not care about the nature of the ML problem; if the supplied metric function supports multiclass classification, then the `MetricFrame` can use it, producing a result for each of the groups identified by the sensitive feature(s). For example, it is perfectly acceptable to pass `sklearn.metrics.confusion_matrix` as the metric function, and supply multiclass data for `y_true` and `y_pred`.

Reply:
I used this, but kept the part about adding thoughts since that is still evolving. Does that work for you?